test/copy-to-s3/copy-to-s3-minio.td

```diff
@@ -1,63 +0,0 @@
-# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
this file was totally unused (looks like an earlier version of `tpch1gb.td` introduced mistakenly in this PR https://github.com/MaterializeInc/materialize/pull/26688)